unittests/CMakeLists.txt

```diff
@@ -47,7 +47,8 @@ target_include_directories( unit_test PUBLIC
                             ${CMAKE_BINARY_DIR}/contracts
                             ${CMAKE_CURRENT_SOURCE_DIR}/contracts
                             ${CMAKE_CURRENT_BINARY_DIR}/contracts
-                            ${CMAKE_CURRENT_BINARY_DIR}/include )
+                            ${CMAKE_CURRENT_BINARY_DIR}/include
+                            ${CMAKE_SOURCE_DIR}/plugins/http_plugin/include )
```

Comment:
Should this line go [here](https://github.com/EOSIO/eos/blob/e1b4f6d5218b8fb72476951af2c02728cbe50da0/tests/CMakeLists.txt#L13)?

Reply:
yes, http_plugin/include  is required for unit-testing function "parse_params" in http_plugin.hpp